server/routes/static.py

```diff
@@ -31,14 +31,17 @@
 
 @bp.route('/')
 def homepage():
-    place_names = place_api.get_display_name('^'.join(_HOMEPAGE_PLACE_DCIDS),
-                                             g.locale)
-    blog_date = babel_dates.format_date(date(2021, 7, 26),
-                                        format='long',
-                                        locale=g.locale)
-    return render_template('static/homepage.html',
-                           place_names=place_names,
-                           blog_date=blog_date)
+    if current_app.jinja_env.globals['DEPLOYMENT'] == "private":
+        return render_template('static/private.html')
+    else:
```

Comment:
else not needed here, since there's a return statement in the if clause above.

Reply:
Done